scripts/apptester/apptester.py

```diff
@@ -251,33 +252,41 @@ def build_applications(self, name=None):
             build_result = [app_data, self.result_matrix[app], 'N/A']
             self.table_data.append(build_result)
 
-    def build_android_java_demo(self):
+
+    def execute_build(self, app_data, file_type):
```

Comment:
I would suggest renaming this method to "execute_remote_build" or something like this.